### DISCUSSION TITLE
Snapdragon and MediaTek OPPO is possible

### DISCUSSION THREAD

**Discussion by NasiGorengBurung** [zenfyrdev/bootloader-unlock-wall-of-shame#17] [Show and tell]
Check in OPPO tab i made changed

https://en.m.wikipedia.org/wiki/Bootloader_unlocking

**Comment by NewVibey** [comment]
Do you think Oppo A53 would be possible to ubl? I need a little bit of help if it's possible.

**Reply by NasiGorengBurung** [reply]
ubl? Stand for unlock bootloader? Right?

**Comment by zenfyrdev** [comment]
Is this the ocdt patching method? Last time I saw this, the script was for Find X6 Pro only.

**Reply by NasiGorengBurung** [reply]
I wish i want to know what OCDT mean

**Reply by NasiGorengBurung** [reply]
There are Snapdragon-powered phone support list in [4] in oppo tab stand for reference
